response_operations_ui/views/case.py

```diff
@@ -8,19 +11,25 @@
     party_controller, survey_controllers
 from response_operations_ui.controllers.case_controller import get_case_events_by_case_id, \
     get_case_by_case_group_id
+from response_operations_ui.controllers.party_controller import get_respondent_by_party_id
 from response_operations_ui.forms import ChangeGroupStatusForm
 
 COMPLETE_STATE = ['COMPLETEDBYPHONE', 'COMPLETE']
 COMPLETED_CASE_EVENTS = ['OFFLINE_RESPONSE_PROCESSED', 'SUCCESSFUL_RESPONSE_UPLOAD', 'COMPLETED_BY_PHONE']
+SUCCESSFUL_CASE_EVENTS = ['OFFLINE_RESPONSE_PROCESSED', 'SUCCESSFUL_RESPONSE_UPLOAD']
```

Comment:
another case event is `ONLINE_QUESTIONNAIRE_RESPONSE` which has the person who _completed_ the EQ, as opposed to the person who launched it

Reply:
I thought that `OFFLINE_RESPONSE_PROCESSED` was the only case event returned from a completed EQ. I can change it to just `COMPLETED_CASE_EVENTS` list in the case their both returned.